ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -7,13 +7,13 @@
 	objects = {
```

Comment:
weirdo XCode stuff - I don't know if this is necessary. Likely not. But... it's XCode.